app/scripts/controllers/membership.js

```diff
@@ -364,5 +363,25 @@ angular
                 }
               });
             });
+
+
+          DataService
+            .list('serviceaccounts', requestContext)
```

Comment:
Don't we need to be getting service accounts from the selected namespace? Won't this only get the service accounts from the current namespace?